src/components/node-list/index.js

```diff
@@ -34,33 +36,35 @@ const NodeListProvider = ({
   nodes,
   nodeSelected,
   tags,
-  types,
+  tagNodeCounts,
+  nodeTypes,
   onToggleNodesDisabled,
   onToggleNodeSelected,
   onToggleNodeActive,
   onToggleParametersActive,
   onToggleTagActive,
   onToggleTagFilter,
   onToggleModularPipelineActive,
-  onToggleModularPipelineFilter,
   onToggleTypeDisabled,
   modularPipelines,
-  sections,
 }) => {
   const [searchValue, updateSearchValue] = useState('');
+
   const items = getFilteredItems({
     nodes,
     tags,
+    nodeTypes,
+    tagNodeCounts,
     modularPipelines,
     nodeSelected,
     searchValue,
   });
 
-  const groups = getGroups({ types, items });
+  const groups = getGroups({ items });
 
   const onItemClick = (item) => {
-    if (isTagType(item.type) || isModularPipelineType(item.type)) {
-      onCategoryItemChange(item, item.checked);
+    if (isGroupType(item.type)) {
```

Comment:
What does `isGroupType` checks for? I've cross referenced this with `node-list-items` and there is no `isGroupType` function declared. Is this check still needed?